osie-runner/run.py

```diff
@@ -81,6 +82,11 @@ def connect_hegel(facility):
     tinkerbell = parse.urlparse(util.value_from_kopt(cmdline_content, "tinkerbell"))
     facility = util.value_from_kopt(cmdline_content, "facility")
 
+handler = logging.handlers.SysLogHandler(
+    address=(f"tinkerbell.{facility}.packet.net", 514)
+)
+log.addHandler(handler)
+
```

Comment:
One thing to take into consideration with this, is this might cause an exception to be raised if the SysLogHandler is unable to connect to the syslog server. A custom handler might need to wrap the sysloghandler in order to capture and dismiss failed remote syslog write attempts.

Reply:
splitting this out into a separate pr